src/scripts/components/structures/Card.js

```diff
@@ -102,6 +103,16 @@ const Footer = styled.div`
   width: 100%;
 `
 
+// Need to recreate this component because there's no rule about the aligment
+
+export const CardStyle = css`
+  background-color: ${props => props.theme.colors.MainCard.background};
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+  padding: ${props => (props.nopadding ? null : CARD_PADDING)};
+`
```

Comment:
I do not think we should be exporting styles/css from components. This is not very standard nor straightforward. If something needs to be shared, it should be its own component that can accept children. 

Component should only be worried about rendering themselves and should not contain logic that will control the appearance of other components

Reply:
This component needs to be refactored.
Let's create a issue for that and change it in another PR.